test/hooks.jl

```diff
@@ -20,6 +20,16 @@ end
 
         @test JuliaSyntax.core_parser_hook(" x \n", "somefile", 1, 0, :statement) == Core.svec(:x,4)
         @test JuliaSyntax.core_parser_hook(" x \n", "somefile", 1, 0, :atom)      == Core.svec(:x,2)
+
+        # https://github.com/JuliaLang/JuliaSyntax.jl/issues/316#issuecomment-1870294857
+        stmtstr =
+            """
+            plus(a, b) = a + b
+
+            # Issue #81
+            f() = nothing
+            """
+        @test JuliaSyntax.core_parser_hook(stmtstr, "somefile", 1, 0, :statement)[2] ∈ 19:21
```

Comment:
What's the range `19:21` for?

Reply:
I don't see that there's a truly unique "right" answer. So it's anything that parses all the code and before the comment starts.

Reply:
Ooh I see :)

If we're choosing a particular somewhat-arbitrary but well-defined behavior I'd prefer to be precise with the test so we can detect any unexpected changes.

Reply:
Done. I went with 19, which is the earliest this can terminate. I believe this probably matches the flisp parser, but not sure.

I wrote a "wide" test initially because of the annoyances I've experienced over time maintaining tests sensitive to what turn out to be irrelevant details (e.g., whether you add a space between elements when printing tuples). So I've sometimes started to ask, "what behavior do I most care about, and what's the widest test I can write that still enforces that behavior?" But I fully understand where you're coming from here, and it probably makes sense given that this is the *origin* of such choices rather than the *receiver* of such choices.